cfme/fixtures/nuage.py

```diff
@@ -58,3 +58,26 @@ def with_nuage_sandbox(networks_provider):
     enterprise = sandbox['enterprise']
     nuage.delete_enterprise(enterprise)
     logger.info('Destroyed sandbox enterprise {} ({})'.format(enterprise.name, enterprise.id))
+
+
+@pytest.fixture(scope='module')
+def with_nuage_sandbox_modscope(appliance, setup_provider_modscope, provider):
+    nuage = provider.mgmt
+    sandbox = create_basic_sandbox(nuage)
+    logger.info('Performing a full refresh, this might take a while.')
+    wait_for(provider.is_refreshed, func_kwargs=dict(refresh_delta=10), timeout=600, delay=10)
+
+    # Check if tenant exists in database, if not fail test immediately
+    tenants_table = appliance.db.client['cloud_tenants']
+    tenant = (appliance.db.client.session
+              .query(tenants_table.name, tenants_table.ems_ref)
+              .filter(tenants_table.ems_ref == sandbox['enterprise'].id).first())
+    assert tenant is not None
```

Comment:
Please include some contextual description here since we're going to hit a setup phase AssertionError here. Something like:

```python
assert tenant is not None, 'Nuage sandbox tenant inventory missing: {}'.format(sandbox)
```